plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -214,6 +217,21 @@ public SqlServerClient(
 
         this.connectorExpressionRewriter = JdbcConnectorExpressionRewriterBuilder.newBuilder()
                 .addStandardRules(this::quoted)
+                .add(new RewriteComparison(ImmutableSet.of(RewriteComparison.ComparisonOperator.EQUAL, RewriteComparison.ComparisonOperator.NOT_EQUAL)))
+                .add(new RewriteIn())
+                .withTypeClass("integer_type", ImmutableSet.of("tinyint", "smallint", "integer", "bigint"))
+                .map("$add(left: integer_type, right: integer_type)").to("left + right")
+                .map("$subtract(left: integer_type, right: integer_type)").to("left - right")
+                .map("$multiply(left: integer_type, right: integer_type)").to("left * right")
+                .map("$divide(left: integer_type, right: integer_type)").to("left / right")
+                .map("$modulus(left: integer_type, right: integer_type)").to("left % right")
+                .map("$negate(value: integer_type)").to("-value")
```

Comment:
Should this not use parenthesis to make sure cases where `value` itself is expression is handled properly?

Seems like this applies to almost every rewrite rule though.

Reply:
you mean, smth like this:
```
 .map("$modulus(left: integer_type, right: integer_type)").to("(left % right)")
                .map("$negate(value: integer_type)").to("(-value)")
```

?

Reply:
Ignore the comment. 

I was thinking of cases where value itself is an expression like `value = a + b` then `$negate(value)` should become `- (a + b)` instead of `- a + b`. But I see today it's not possible.